### ISSUE TITLE
Kr8s should read server CRTs from kubeconfig folder

### ISSUE THREAD

**Issue by aschoeke-rtr** [kr8s-org/kr8s#305]
### Which project are you reporting a bug for?

kr8s

### What happened?

When running `_ = await api.version() ` I receive an error message with a lenghty traceback (that I can provide on request).

```python
  File "/home/xxxxxxxxxxxxx/.pyenv/versions/3.11.6/lib/python3.11/ssl.py", line 979, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1006)
```
At first I thought it was an issue on my side. However, `kubectl` works fine and no `skip-tls-verify` is present in the kubeconfig. The server CRTs are present in the kube folder, referenced in the config file and therefore should be used for SSL verification.

Further more, the command succeeds if the crt-file is explicitly referenced with the env setting `SSL_CERT_FILE=~/.kube/cluster-ca.crt`.

### Anything else?

Most likely related to this PR: https://github.com/kr8s-org/kr8s/pull/304

**Comment by benedikt-bartscher** [comment]
@aschoeke-rtr you are correct, that's why I ended up writing the PR yesterday. Are you using a minikube cluster?

**Comment by aschoeke-rtr** [comment]
No, to my knowledge it is a standard k8s cluster.

**Comment by jacobtomlinson** [comment]
Thanks for raising this. I agree this should be closed by the bug fix @benedikt-bartscher raised in #304. Just need a test over there and it should be good to merge.
